File: sprint SYMFONY/webApplication/src/ReclamationBundle/Entity/Reclamation.php
<?php

namespace ReclamationBundle\Entity;

use Doctrine\ORM\Mapping as ORM;

/**
 * Reclamation
 *
 * @ORM\Table(name="reclamation", indexes={@ORM\Index(name="user", columns={"idUser"})})
 * @ORM\Entity
 */
class Reclamation
{
    /**
     * @var integer
     *
     * @ORM\Column(name="idRec", type="integer", nullable=false)
     * @ORM\Id
     * @ORM\GeneratedValue(strategy="IDENTITY")
     */
    public $idrec;

    /**
     * @var string
     *
     * @ORM\Column(name="description", type="string", length=20, nullable=false)
     */
    public $description;

    /**
     * @var integer
     *
     * @ORM\Column(name="etat", type="integer", nullable=false)
     */
    public $etat;

    /**
     * @var \DateTime
     *
     * @ORM\Column(name="dateRec", type="date", nullable=false)
     */
    public $daterec;

    /**
     * @var string
     *
     * @ORM\Column(name="reponse", type="text", length=65535, nullable=true)
     */
    public $reponse;

    /**
     * @var string
     *
     * @ORM\Column(name="typeRec", type="string", length=255, nullable=false)
     */
    public $typerec;

    /**
     * @var \User
     *
     * @ORM\ManyToOne(targetEntity="AppBundle\Entity\User")
     * @ORM\JoinColumns({
     *   @ORM\JoinColumn(name="idUser", referencedColumnName="idUser")
     * })
     */
    public $user;

    /**
     * @return int
     */
    public function getIdrec()
    {
        return $this->idrec;
    }

    /**
     * @param int $idrec
     */
    public function setIdrec($idrec)
    {
        $this->idrec = $idrec;
    }

    /**
     * @return string
     */
    public function getDescription()
    {
        return $this->description;
    }

    /**
     * @param string $description
     */
    public function setDescription($description)
    {
        $this->description = $description;
    }

    /**
     * @return int
     */
    public function getEtat()
    {
        return $this->etat;
    }

    /**
     * @param int $etat
     */
    public function setEtat($etat)
    {
        $this->etat = $etat;
    }

    /**
     * @return \DateTime
     */
    public function getDaterec()
    {
        return $this->daterec;
    }

    /**
     * @param \DateTime $daterec
     */
    public function setDaterec($daterec)
    {
        $this->daterec = $daterec;
    }

    /**
     * @return string
     */
    public function getReponse()
    {
        return $this->reponse;
    }

    /**
     * @param string $reponse
     */
    public function setReponse($reponse)
    {
        $this->reponse = $reponse;
    }

    /**
     * @return string
     */
    public function getTyperec()
    {
        return $this->typerec;
    }

    /**
     * @param string $typerec
     */
    public function setTyperec($typerec)
    {
        $this->typerec = $typerec;
    }

    /**
     * @return \User
     */
    public function getUser()
    {
        return $this->user;
    }

    /**
     * @param \User $user
     */
    public function setUser($user)
    {
        $this->user = $user;
    }


}

File: sprint JAVA/desktopApplication/src/entity/reclamation/Reclamation.java
/*
 * To change this license header, choose License Headers in Project Properties.
 * To change this template file, choose Tools | Templates
 * and open the template in the editor.
 */
package entity.reclamation;

import entity.user.User;
import java.sql.Date;

/**
 *
 * @author Legion
 */
public class Reclamation {
    private int idRec ;
    private User user ;
    private String description ;
    private int etat ;
    private Date dateRec ;
    private String reponse ;
    private TypeReclamation tr;
    

    public Reclamation() {
    }

    public Reclamation(int idRec, User user, String description, int etat, Date dateRec, String reponse,TypeReclamation tr ) {
        this.idRec = idRec;
        this.user = user;
        this.description = description;
        this.etat = etat;
        this.dateRec = dateRec;
        this.reponse = reponse;
        this.tr=tr;
        
    }

    public int getIdRec() {
        return idRec;
    }

    public void setIdRec(int idRec) {
        this.idRec = idRec;
    }

    public User getUser() {
        return user;
    }

    public void setUser(User user) {
        this.user = user;
    }

    

    public String getDescription() {
        return description;
    }

    public void setDescription(String description) {
        this.description = description;
    }

    public int getEtat() {
        return etat;
    }

    public void setEtat(int etat) {
        this.etat = etat;
    }

    public Date getDateRec() {
        return dateRec;
    }

    public void setDateRec(Date dateRec) {
        this.dateRec = dateRec;
    }

    public String getReponse() {
        return reponse;
    }

    public void setReponse(String reponse) {
        this.reponse = reponse;
    }

    public TypeReclamation getTr() {
        return tr;
    }

    public void setTr(TypeReclamation tr) {
        this.tr = tr;
    }

    @Override
    public String toString() {
        return "Reclamation{" + "idRec=" + idRec + ", user=" + user + ", description=" + description + ", etat=" + etat + ", dateRec=" + dateRec + ", reponse=" + reponse + ", tr=" + tr + '}';
    }
    
    
    
}
